wagtail/wagtailcore/tests/test_widgets.py

```diff
@@ -0,0 +1,40 @@
+from django.test import TestCase
+from wagtail.wagtailcore.fields import BaseTextAreaWidget, RichTextArea
+
+
+class TestBaseTextAreaWidget(TestCase):
+
+    def test_get_panel_implemented(self):
+        with self.assertRaises(NotImplementedError):
+            widget = BaseTextAreaWidget()
+            widget.get_panel()
+
+    def test_render_js_init(self):
+        widget = BaseTextAreaWidget()
+        js_init = widget.render_js_init('test-id', 'test', None)
+        # self.assertIsInstance(js_init, unicode) Travis CI - flake8 does not recognise unicode
+        self.assertEqual(len(js_init), 0)
+
+
+class TestRichTextAreaWidget(TestCase):
+
+    def test_get_panel_implemented(self):
+        from wagtail.wagtailadmin.edit_handlers import RichTextFieldPanel
+        widget = RichTextArea()
+        panel_proxy = widget.get_panel()
+        panel = panel_proxy(None)
+        self.assertIsInstance(panel, RichTextFieldPanel)
+
+    def test_widget_default_render_js_init(self):
+        widget = RichTextArea()
+        js_init = widget.render_js_init('test-id', 'test', None)
+        # self.assertIsInstance(js_init, unicode) Travis CI - flake8 does not recognise unicode
```

Comment:
`test_widget_default_render_js_init` and `test_widget_eit_config_render_js_init` looks good for me.

If you want to check unicode in Python 2 and 3, you need to use `six.text_type`, but I think we can omit this check in these two tests.